oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java

```diff
@@ -175,20 +173,18 @@ private static long tsToMilli(Timestamp timestamp) {
         return "oc";
     }
 
-    @Override public void active(List<String> enabledRules,
-        MeterSystem service, GRPCHandlerRegister grpcHandlerRegister) {
-        List<Rule> rules;
-        try {
-            rules = Rules.loadRules("otel-oc-rules", enabledRules);
-        } catch (ModuleStartException e) {
-            log.warn("failed to load otel-oc-rules");
-            return;
-        }
+    @Override
+    public void active(
+        List<String> enabledRules,
+        MeterSystem meterSystem,
+        GRPCHandlerRegister grpcHandlerRegister)
+        throws ModuleStartException {
+
+        final List<Rule> rules = Rules.loadRules("otel-oc-rules", enabledRules);
```

Comment:
So, all MAL scripts are still going to be located in `otel-oc-rules`? Should we rename the folder to `otel-rules`?

Reply:
We can do that.

Reply:
Let's do this, but note we need to adjust some docs about the folder name including docs and package config files.

Reply:
@yswdqz Notice this. I prefer I can merge this PR first, then you should adjust your PR accordingly.

Reply:
@kezhenxu94 We need a breaking change about the folder name change.

Reply:
> @yswdqz Notice this. I prefer I can merge this PR first, then you should adjust your PR accordingly.

Got it.